package.json

```diff
@@ -189,6 +191,7 @@
     "@babel/core": "^7.8.4",
     "@babel/preset-env": "^7.6.0",
     "@babel/runtime": "^7.8.4",
+    "@react-native-community/toolbar-android": "^0.1.0-rc.2",
```

Comment:
Note sure if we need toolbar-android, added this to fix a compilation error related to react-native-vector-icons

Reply:
What compilation error are you seeing? Nobody else has brought up issues